src-electron/db/query-zcl.js

```diff
@@ -449,8 +452,7 @@ async function selectAllStructItemsById(db, id) {
 SELECT
   STRUCT_ITEM.FIELD_IDENTIFIER,
   STRUCT_ITEM.NAME,
-  (SELECT DATA_TYPE.NAME FROM DATA_TYPE WHERE DATA_TYPE.DATA_TYPE_ID = STRUCT_ITEM.DATA_TYPE_REF) AS TYPE,
```

Comment:
I made this change which is slightly unrelated & more of a cleanup, but makes my changes clearer

i.e. I don't understand why there was a sub-query here to select `DATA_TYPE.NAME`, when we were already joining with `DATA_TYPE` & selecting `DATA_TYPE.NAME` -> `dataTypeReferenceName` (which was completely unused but would have the same value)